orm/uint64_index.go

```diff
@@ -40,6 +44,11 @@ func (i UInt64Index) Get(ctx HasKVStore, searchKey uint64) (Iterator, error) {
 	return i.multiKeyIndex.Get(ctx, EncodeSequence(searchKey))
 }
 
+// GetPaginated returns a result iterator for the searchKey based on pageRequest.
```

Comment:
```suggestion
// GetPaginated creates an iterator, starting from the searchKey and based on pageRequest.
```

Reply:
Well it's not necessarily "starting" from the `searchKey`, but more the combined `searchKey` and `pageRequest.Key` if provided. I have updated the comment to make it clearer, wdyt?

Reply:
good, thanks.